compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -114,15 +114,10 @@ enum class VectorPreProcStrategy {
   None
 };
 
-// NOTE: This flag is meant for testing + experimentation and should not be
-// used in deployment.
-static llvm::cl::opt<bool> clExperimentalArmForceSSVE(
-    "iree-experimental-llvmcpu-arm-force-ssve",
-    llvm::cl::desc(
-        "Controls whether to disable SME tiling when SME+SSVE are enabled "
-        "with +sme. As a result, IREE will effectively target SSVE "
-        "instead of SME. This flag is experimental and should only be "
-        "used for testing."),
+static llvm::cl::opt<bool> clDisableArmSMETiling(
```

Comment:
Double negative is confusing. So this is enabling sme tiling by default?

Reply:
I was just following the naming used by other flags, e.g. `iree-llvmcpu-disable-distribution`,  `iree-llvmcpu-disable-vector-peeling`. So with this flag tiling for ArmSME is enabled unless you pass `--iree-llvmcpu-disable-arm-sme-tiling`.

Reply:
My understanding is that if something is opt-in (i.e. not on by default) you have a `enable` flag, and if something is opt-out (i.e. on by default) you have a `disable` flag.

Reply:
I also don't quite see how `clDisableArmSMETiling` is a double negative 😅 And yeah, this is similar to e.g. `clDisableDistribution`:
https://github.com/iree-org/iree/blob/5145f7e4e9ab65de1dd91a015440526af678b826/compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp#L54-L57